section4/exercises/burrito.rb

```diff
@@ -11,9 +11,25 @@ def initialize(protein, base, toppings)
     @base     = base
     @toppings = toppings
   end
+  def add_toppings(item)
+    @toppings << item
+    puts "Your #{item} have been added"
+  end
+
+  def remove_toppings(item)
+     puts "Your #{item} have been removed"
+  end
+
+  def change_protein(protein)
+    puts "Your protein will be changed to #{protein}"
```

Comment:
Same here- don't forget some logic that actually does the thing you report back has happened. You will need to alter the properties at play.